addOns/webuipoc/src/main/pocs/reactWebUI/package.json

```diff
@@ -4,26 +4,28 @@
   "private": true,
   "homepage": ".",
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@mui/icons-material": "^5.15.18",
+    "@mui/material": "^5.15.18",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.6.8",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-scripts": "^5.0.1"
+
```

Comment:
Can these empty lines not be added? Same in other places.

Reply:
Removed

Reply:
It's not only this line.

Reply:
Removed from package.json